src/change/sdp.rs

```diff
@@ -401,6 +401,37 @@ impl<'a> SdpApi<'a> {
             None
         }
     }
+
+    /// Combines the modifications made in [`SdpApi`] with those in [`SdpPendingOffer`].
+    ///
+    /// This function merges the changes present in [`SdpApi`] with the changes
+    /// in [`SdpPendingOffer`]. In result this [`SdpApi`] will incorporate modifications
+    /// from both the previous [`SdpPendingOffer`] and any newly added changes.
+    ///
+    /// ## Example
+    ///
+    /// ```no_run
+    ///
+    /// # use str0m::media::{Direction, MediaKind};
+    /// # use str0m::Rtc;
+    ///
+    /// let mut rtc = Rtc::new();
```

Comment:
Remove the whitespace lines before and after the import lines, or we get weird gaps in the generated doc:

<img width="1035" alt="image" src="https://github.com/algesten/str0m/assets/227204/07185f06-fee1-4b4b-b8a5-a378a609c722">

Reply:
@algesten ,

Fixed!

Reply:
Great!